swift/extractor/SwiftExtractor.cpp

```diff
@@ -198,10 +199,10 @@ void codeql::extractSwiftFiles(SwiftExtractorState& state, swift::CompilerInstan
         continue;
       }
       archiveFile(state.configuration, *sourceFile);
-      encounteredModules = extractDeclarations(state, compiler, *module, sourceFile);
+      encounteredModules = extractDeclarations(state, compiler, *module, sourceFile, nullptr);
```

Comment:
maybe an inline comment for what `nullptr` is (in case someone is reading without a good IDE :wink: )